tests/dependencies/updater_test.php

```diff
@@ -608,37 +608,17 @@ public function testUpdateWithBrokenJson()
         );
 
         ob_start();
-        $result = $dependencies->update();
+        $result = $dependencies->updateAll();
         $output = ob_get_clean();
 
         $this->assertTrue($result->isErr());
         $this->assertEquals('missing sources', $result->getErr());
         $this->assertEquals("[Updating themes/my-theme]\n", $output);
     }
 
-    public function testUpdateDependencyWithNoLock()
+    public function testUpdateSingleWithNoLock()
     {
         $dir = $this->getDir();
-        $whippetJson = $this->getWhippetJson([
-            'src' => [
-                'plugins' => 'git@git.dxw.net:wordpress-plugins/',
-            ],
-            'themes' => [
-                [
-                    'name' => 'my-theme',
-                    'ref' => 'v1.4',
-                ],
-            ],
-            'plugins' => [
-                [
-                    'name' => 'my-plugin',
-                    'ref' => 'v1.6',
-                ],
-            ],
-        ]);
-        $this->addFactoryCallStatic('\\Dxw\\Whippet\\Files\\WhippetJson', 'fromFile', $dir.'/whippet.json', \Result\Result::ok($whippetJson));
-
-        file_put_contents($dir.'/whippet.json', 'foobar');
```

Comment:
The refactoring affected the order of events - previously, whippet.json was loaded before whippet.lock, so the test for what happens in the absence of whippet.lock needed a whippet.json file to load successfully first.

That no longer happens - `updateSingle()` check for the whippet.lock file before anything else, so the whippet.json created for this test isn't needed.

I've added a note to the appropriate commit message.